# [pytest] section in setup.cfg files is no longer supported
# TODO: Upstream

--- setup.cfg.orig	2020-01-13 04:27:33 UTC
+++ setup.cfg
@@ -4,7 +4,7 @@ optimize = 1
 [sdist]
 formats = bztar,zip
 
-[pytest]
+[tool:pytest]
 norecursedirs = .git .hg .svn build dist
 
 [egg_info]
